examples/50_hopper_gemm_with_epilogue_swizzle/50_hopper_gemm_with_epilogue_swizzle.cu

```diff
@@ -485,7 +485,7 @@ int main(int argc, char const **args) {
   // Tiled copy from Smem to Registers
   // Note : CuTe will vectorize this copy if the tiling + swizzling above were right
   using TiledCopyS2R = TiledCopy<
-                         Copy_Atom<DefaultCopy, ElementAcc>,
+                         Copy_Atom<AutoVectorizingCopyWithAssumedAlignment<128>, ElementAcc>,
```

Comment:
Since the DefaultCopy used to be `AutoVectorizingCopyWithAssumedAlignment<128>`, it must now be changed to explicitly specify this to preserve behavior. I believe this is the easiest way to reduce the scope of this PR and keep changes minimal.